cranelift/codegen/meta/src/shared/instructions.rs

```diff
@@ -2651,6 +2651,20 @@ pub(crate) fn define(
         .operands_out(vec![a]),
     );
 
+    ig.push(
+        Inst::new(
+            "bswap",
+            r#"
+        Reverse the byte order of an integer.
+
+        Reverses the bytes in ``x``.
+        "#,
+            &formats.unary,
+        )
+        .operands_in(vec![x])
```

Comment:
I'm wondering if we should exclude 8-bit integers for this instruction. There's no sensible implementation of byte-swapping when there's only one byte.

Reply:
Good idea, excluded 8-bit and also 128-bit since the latter is unimplemented for now